packages/helpers/src/components/iframe-runtime.tsx

```diff
@@ -74,17 +76,19 @@ export const IframeRuntime: React.FC<IProps> =
         }
         return Promise.resolve({});
       });
+
       phone.addListener("interactiveState", (newInteractiveState: any) => {
         setInteractiveStateRef.current?.(newInteractiveState);
         if (flushOnSave) {
           // don't wait the default 2000ms timeout before saving
           flushStateUpdates();
         }
-        if (onUnloadCallback && resolveOnUnload.current) {
+
+        if (onUnloadCallbackRef?.current && resolveOnUnload.current) {
```

Comment:
I think you can change this to `onUnloadCallbackRef.current` as the current value is always set on line 56 (the current value may be undefined but the ref is always defined).